src/cnxml/index.ts

```diff
@@ -14,6 +14,8 @@ export type CnxmlVersion = '0.7' | '0.8'
 
 /** A serialized CNXML document */
 export interface Document {
+    /** Class name for the <document> element */
+    className?: string
```

Comment:
While `Document` is not a Slate node, handling of classes on document should be compatible with `WithClasses`. This means that this field should be `classes: string[]`.